app/javascript/app/pages/regions/sectoral-circumstances/forestry/forestry-selectors.js

```diff
@@ -0,0 +1,129 @@
+import { getTranslate } from 'selectors/translation-selectors';
+import { getProvince } from 'selectors/provinces-selectors';
+import { createSelector, createStructuredSelector } from 'reselect';
+import camelCase from 'lodash/camelCase';
+import upperFirst from 'lodash/upperFirst';
+import { getThemeConfig, getTooltipConfig } from 'utils/graphs';
+
+const FORESTRY_CODE = 'forest_cover_loss';
+
+const CHART_COLORS = [ '#FC7E4B', '#00B4D2' ];
+
+const getIndicators = ({ indicators }) => indicators && indicators.data;
+
+const getForestryIndicators = createSelector(getIndicators, indicators => {
+  if (!indicators) return null;
+
+  return indicators.values &&
+    indicators.values.filter(ind => ind.indicator_code === FORESTRY_CODE);
+});
+
+const getForestryIndicatorsPerProvince = createSelector(
+  [ getForestryIndicators, getProvince ],
+  (forestryIndicators, provinceIso) => {
+    if (!forestryIndicators || !provinceIso) return null;
+
+    return forestryIndicators.filter(
+      ind => ind.location_iso_code3 === provinceIso
+    );
+  }
+);
+
+const getUniqueYears = createSelector(
+  getForestryIndicatorsPerProvince,
+  data => {
+    if (!data) return null;
+
+    const allYears = data
+      .map(d => d.values)
+      .map(arr => arr.map(o => o.year))
+      .flatten();
```

Comment:
I didn't know there is `flatten` in Array prototype.

Reply:
About #mooshgate (backwards incompatible use of `flatten`)
https://developers.google.com/web/updates/2018/03/smooshgate

Reply:
What!? `flat` is flattening one level by default. That would cause a lot of misunderstanding. Why they just can't add `flatten` with default flattening all levels as this is more common name and desired behavior.

Reply:
Btw. as this is still experimental, better to use lodash `flatten`